### PR TITLE
Feature: Trim dark and light for login bg images, solves #192.

### PR DESCRIPTION
**Pull request by lucaboesch**

### CODE REVIEW

File: locallib.php [hunk 1]
```diff
@@ -647,14 +647,14 @@ function theme_boost_union_get_loginbackgroundimage_text() {
...
+            if (strcmp($filename, trim($settings[0])) == 0) {
                 // If the color value is not acceptable, replace it with dark.
-                if ($settings[2] != 'dark' && $settings[2] != 'light') {
+                if (trim($settings[2]) != 'dark' && trim($settings[2]) != 'light') {
                     $settings[2] = 'dark';
                 }
 
                 // Return the text + text color that belongs to the randomly selected image.
-                return array(format_string($settings[1]), $settings[2]);
+                return array(format_string(trim($settings[1])), trim($settings[2]));
```
Comment (abias) [inline locallib.php:657]: You are trimming the same string three times in a row. This is not really efficient. You should do it once and store the result back to the array. 

I will change that myself before merging the PR.

**Review by abias — CHANGES_REQUESTED** [review]
Thank you for working on this, Luca!

I found two things which should be changed before merging this PR. Both are minor things, but we strive for clean solutions, don't we?

**Comment by lucaboesch** [comment]
Yes, sure, @abias I agree fully with both (the amendments and that we go for clean solutions). Please proceed as proposed.

**Review by abias — APPROVED** [review]

**Comment by abias** [comment]
The tests have passed again, so I will merge this now.

**Pull request merged**